libs/ui/src/lib/tabs/tabs.component.stories.ts

```diff
@@ -65,3 +67,62 @@ Mobile.parameters = {
     defaultViewport: 'iphone12mini',
   },
 };
+
+const TemplateWithIcon: Story<TabComponent> = (args: TabComponent) => ({
+  props: args,
+  template: `
+      <cvi-ng-tabs (activeTabChange)="activeTabChange($event)">
+        <cvi-ng-tab
+            title="Tab 1"
+            [iconName]="iconName"
+            [svgClass]="'svg-class'"
+            [iconPosition]="iconPosition"
+            [iconHeight]="iconHeight"
+            [gap]="gap"
+            >{{ content }}</cvi-ng-tab
+  >
+        <cvi-ng-tab title="Tab 2, with a form">
+          <div [formGroup]="form">
```

Comment:
Because the form variable is there in the template but it is not actually set for the template like it is done in the default `Template`, there's an error in the console of the Storybook: 

```
ERROR Error: NG01052: formGroup expects a FormGroup instance. Please pass one in.
```

I'd suggest to simply remove the form from this story. The goal of the With icon story is simply to illustrate how tab icon can be added and what it looks like. The form example is not needed.